src/vs/workbench/contrib/debug/common/debugSource.ts

```diff
@@ -134,7 +134,7 @@ export function getUriFromSource(raw: DebugProtocol.Source, path: string | undef
 		if (typeof raw.sourceReference === 'number' && raw.sourceReference > 0) {
 			return URI.from({
 				scheme: DEBUG_SCHEME,
-				path,
+				path: path?.replace(/^\/+/g, '/'), // #174054
```

Comment:
Do you think there's a legit reason the DA would send a path formed like this? To me this feels like trying to hide a problem

Reply:
UNC paths, I would wager

in DAP however we don't specify anything about the path, so as a client VS Code should accept any string. (There's probably more cases we don't handle come to think of it)